tap_csv/tests/test_core.py

```diff
@@ -69,3 +69,21 @@ def test_standard_tap_tests_csv_dialect():
     tests = get_standard_tap_tests(TapCSV, config=SAMPLE_CONFIG)
     for test in tests:
         test()
+
+# Run standard built-in tap tests from the SDK, with metadata columns included:
+def test_standard_tap_tests():
```

Comment:
```suggestion
def test_standard_tap_tests_metadata_cols():
```

@TyShkan this had a conflicting name with another test so it was getting skipped.